sale_line_refund_to_invoice_qty/wizards/account_move_reversal.py

```diff
@@ -1,18 +1,28 @@
 # Copyright 2021 ForgeFlow (http://www.forgeflow.com)
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class AccountMoveReversal(models.TransientModel):
     _inherit = "account.move.reversal"
 
     sale_qty_to_reinvoice = fields.Boolean(
         string="This credit note will be reinvoiced",
-        default="True",
+        default=lambda self: self.company_id.reinvoice_credit_note_default,
         help="Leave it marked if you will reinvoice the same sale order line "
         "(standard behaviour)",
     )
 
+    @api.model
+    def default_get(self, flds):
```

Comment:
```suggestion
    def default_get(self, fields):
```

sorry for the nitpicking 🙏🏼

Reply:
**question**: I renamed the argument to `fields` because it shadows `odoo.fields` that we import at the top of the file. Is it absolutely necessary to leave the name as `fields`?

Reply:
I see, Odoo uses 'fields' anyway, but maybe better to rename it. However I would prefer maybe 'fields_list', what do you think?

Reply:
Yeah, it fits nicely. Thanks.